immuneML/workflows/instructions/TrainMLModelInstruction.py

```diff
@@ -118,7 +118,7 @@ class TrainMLModelInstruction(Instruction):
             number_of_processes: 4 # number of parallel processes to create (could speed up the computation)
             optimization_metric: balanced_accuracy # the metric to use for choosing the optimal model and during training
             refit_optimal_model: False # use trained model, do not refit on the full dataset
-            store_encoded_data: True # store encoded datasets in pickle format
+            store_encoded_data: True # store encoded datasets in binary format
```

Comment:
maybe add (ImmuneML format) at the end, so people know how it can be imported if necessary

Reply:
do you agree with removing this option completely? if one would want to export the encoded data, they can do so using DesignMatrixExporter and then get csv file. This format is not useful anyway (cannot import elsewhere)

Reply:
updated as well. it should not affect galaxy, but if it does, it is a matter of removing the parameter (not visible to users anyway)

Reply:
ah yes I didnt read this message earlier, but I think it is fine either way with or without the option, just nice if the metadata file is stored to know which repertoires were in which split.